Indico/Mutation/WorkflowSubmission.cs

```diff
@@ -25,12 +25,14 @@ public class WorkflowSubmission : WorkflowSubmissionBase, Mutation<List<int>>
 
         public async Task<List<int>> Exec()
         {
-            JObject response = await base.Exec();
-            JArray ids = (JArray)response.GetValue("submissionIds");
+            var response = await base.Exec();
+            var ids = (JArray)response.GetValue("submissionIds");
 
-            List<int> submissionIds = new List<int>();
-            foreach (JToken submissionId in ids)
+            var submissionIds = new List<int>();

```

Comment:
could probably be a linq expression here

Reply:
It can be. I followed all the rules from .editorconfig. I might be wrong but isn't this kind of change suggested by R#?

Reply:
I fixed that, however, I couldn't find any option in `.editorconfig` file to suggest this.